integrationtest/elasticsearch/src/test/java/org/hibernate/search/elasticsearch/test/PurgeIT.java

```diff
@@ -83,18 +80,7 @@ public void multiplePurges() throws Exception {
 		assertEquals( "Wrong total number of entries", 3, all.size() );
 
 		// Expect 0 failure in the backend threads
-		logged.expectEventMissing( new TypeSafeMatcher<LoggingEvent>() {
-
-			@Override
-			public void describeTo(Description description) {
-				description.appendText( "a LoggingEvent with ERROR level or higher" );
-			}
-
-			@Override
-			protected boolean matchesSafely(LoggingEvent item) {
-				return item.getLevel().isGreaterOrEqual( Level.ERROR );
-			}
-		} );
+		logged.expectLevelMissing( Level.ERROR );
```

Comment:
Actually Infinispan uses `org.infinispan.query.helper.StaticTestingErrorHandler` and the test helpers make sure to inject it consistently in almost all tests.